example_configs/podman-quadlets/lldap.container

```diff
@@ -0,0 +1,71 @@
+###
+### lldap - podman quadlet file
+### https://docs.podman.io/en/latest/markdown/podman-systemd.unit.5.html
+###
+
+[Unit]
+Description=LLDAP container
+Documentation=https://github.com/lldap/lldap
+Requires=lldap-db.service
+
+[Service]
+Restart=on-failure
+
+[Container]
+AutoUpdate=registry
+ContainerName=lldap
+Image=docker.io/lldap/lldap:stable-debian
+
+# using multiple networks for traefik and db, not essential
+Network=frontend.network
+Network=backend.network
+
+# environmental variables
+Environment=UID=1000
+Environment=GID=1000
+Environment=LLDAP_LDAP_BASE_DN=dc=example,dc=com
+Environment=LLDAP_DATABASE_URL=postgres://lldapuser:lldappass@lldap-db/lldap
+Environment=LLDAP_LDAP_USER_EMAIL=admin@example.com
+
+# set secrets in files, see below for environmental variable method
+Environment=LLDAP_KEY_SEED_FILE=/run/secrets/lldap-key-seed
+Environment=LLDAP_JWT_SECRET_FILE=/run/secrets/lldap-jwt-secret
+Environment=LLDAP_LDAP_USER_PASS_FILE=/run/secrets/lldap-ldap-user-pass
+
+# bootstrap.sh environmental variables
+# Environment=LLDAP_URL=http://localhost:17170
+# Environment=LLDAP_ADMIN_USERNAME=admin
+# Environment=LLDAP_ADMIN_PASSWORD_FILE=/run/secrets/lldap-ldap-user-pass
+
+# health check
+HealthCmd=/app/lldap healthcheck
+HealthInterval=5s
```

Comment:
Every 5s is a bit intense, no? Maybe 30s is enough as a default

Reply:
i based on kubernetes' default health checks, i have scaled it back to every 30 seconds!